src/main.js

```diff
@@ -1,8 +1,16 @@
-import Vue from 'vue'
-import App from './App.vue'
 
-Vue.config.productionTip = false
+import Vue from "vue";
+import App from "./App.vue";
+import BootstrapVue from 'bootstrap-vue'
+import 'bootstrap/dist/css/bootstrap.css'
+import 'bootstrap-vue/dist/bootstrap-vue.css'
+import lodash from 'lodash';    
+Object.defineProperty(Vue.prototype, '$lodash', { value: lodash });
```

Comment:
why do we need lodash?

Reply:
because i am manipulating array of users to sort them and found many of examples that are using Lodash so that i have  tried also and it worked in my case that's why i have used Lodash to sort the array